### DISCUSSION TITLE
Issues to install on MacOS Ventura

### DISCUSSION THREAD

**Discussion by michalw2-alt** [noborus/ov#268] [General]
Hi,

I have downloaded and extracted https://github.com/noborus/ov/archive/refs/tags/v0.14.0.tar.gz and I'm getting following error while trying to install ov on my M1 Mac.
I would rather not use mac ports or homebrew.

I'm running command below in the folder where tar.gz was extracted.
This command is according to documentation: https://github.com/noborus/ov#binary

sudo install ov /usr/local/bin
Password:
install: ov: Operation not permitted

Could you please advise what could possible wrong I am doing?
Thank you 

#Just figured it out (needed to run command in native Terminal)

**Comment by noborus** [comment]
Are you sure this is resolved?
I'm always open to pull requests to improve the documentation.
